tensorflow/compiler/jit/compilability_check_util.cc

```diff
@@ -265,7 +265,10 @@ bool RecursiveCompilabilityChecker::OpIsSlow(const Node& node) const {
   // b/135640736: MatrixInverse performance issues.
   return node.type_string() == "SelfAdjointEigV2" ||
          node.type_string() == "Svd" || node.type_string() == "Qr" ||
-         node.type_string() == "MatrixInverse";
+         node.type_string() == "MatrixInverse" ||
+         node.type_string() == "ResizeNearestNeighbor" ||
```

Comment:
Normally we file a bug internally with some details on why the op is slow.  Since you can't file an internal bug do you mind adding some comments on why these ops are slow / undesirable?

Reply:
To be precise, I am opting out these ops, no because they are slow, but because they create convolutions that CuDNN can't handle.  I looked around for lists similar to OpIsSlow, but didn't find anything more proper.  Please let me if there is a better way, or whether to create another list.

I have explained how the error is triggered in my first comment in this PR.  I am attaching a python code reproducing the error.  You can switch on/off XLA to observe the difference.  Hopefully it is sufficient for a bug description.

import tensorflow as tf
import numpy as np

image = tf.placeholder(tf.float32, shape=[16, 256, 256, 16], name='image')
resize_nearest_neighbor = tf.image.resize_images(image, size=[512,512], method=tf.image.ResizeMethod.NEAREST_NEIGHBOR, align_corners=True)
feed_dict={image: np.random.random_sample([16, 256, 256, 16])}

sess = tf.Session()
with sess.as_default():
    actual_resize_nearest_neighbor = resize_nearest_neighbor.eval(feed_dict)

Reply:
Sorry, I meant comments in the code.

Although I guess just liking to this discussion thread would be fine too.

Reply:
@sanjoy, I have added comments that link to this thread.
BTW, I suppose that users can switch cluster/allow_slow_ops at run-time.  Can you show me how to do that?
P.S. I will make merge to push forward https://github.com/tensorflow/tensorflow/pull/30336, as I don't change image_resize_ops.cc.  I will need your help to review it again.

Reply:
> @sanjoy, I have added comments that link to this thread.

Thanks!

> BTW, I suppose that users can switch cluster/allow_slow_ops at run-time. Can you show me how to do that?

TF users can't change these options directly.  These bits are mainly used to control the behavior of `RecursiveCompilabilityChecker` by the various clients of the class (none of which are directly user facing).

> P.S. I will make merge to push forward #30336, as I don't change image_resize_ops.cc. I will need your help to review it again.

Reply:
Is there any further comment for me to address?  If not, please approve the PR.

Reply:
I already approved it the last time, but it looks like a CI build failed.  I'll re-trigger the CI.